src/components/ListItem.jsx

```diff
@@ -59,20 +60,26 @@ export function ListItem({
 		}
 	};
 
+	const urgencyClass = sortCriteria.tag.toLowerCase().replace(/\s/g, '');
```

Comment:
You are using regex to convert your initial tags ("No longer active", "Overdue", "Due soon", etc) to css class friendly name. I would say this step is not actually necessary, and you can just set the tags and css classes to be the same exact text. So for example, instead of setting the sortCriteria tag to "No longer active" in your `comparePurchaseUrgency` function, you can set it directly to "no-longer-active" (kebab case or camel case, or all lowercase, whatever you choose). Then in your ListItem you can just get the `sortCriteria.tag` directly without needing to convert it to the format you want..

Reply:
Thank you, @jtkabenni As of now, changing the tag content would reflect on the UI. I will check in with Nick to modify in the future. Thank you again!